File: Programming basics/Homework 2 Data-Types-And-Variables/09.ExchangeVariableValues/ExchangeVariableValues.cs
using System;

namespace _09.ExchangeVariableValues
{
    class ExchangeVariableValues
    {
        static void Main()
        {
            int a = 5;
            int b = 10;
            int temp = new int();
            Console.WriteLine("Before: ");
            Console.WriteLine("a = {0}",a);
            Console.WriteLine("b = {0}",b);

            //Exchange
            temp = b;
            b = a;
            a = temp;
            Console.WriteLine("After: ");
            Console.WriteLine("a = {0}", a);
            Console.WriteLine("b = {0}", b);
        }
    }
}

File: 4.1 C Programming/Homework 2 C Data Types/ExchangeVariableValues.c


#include <stdio.h>
#include <stdlib.h>

/*
 */ 
int main(int argc, char** argv) {
    int a = 5;
    int b = 10;
    
    printf("Before a = %d, b = %d\n\n",a,b);
    
    int temp = a;
    a = b;
    b = temp;
    
    printf("After a = %d, b = %d\n",a,b);
    return (EXIT_SUCCESS);
}

